pages/index.js

```diff
@@ -1,3 +1,27 @@
+import styled from "styled-components";
+import FoodCard from "../components/FoodCard";
+import useSWR from "swr";
+import { fetcher } from "../helpers/api";
+
 export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+  const { data: data, error } = useSWR("/api/recipes", fetcher);
+
+  if (error) return <h1>There was an error</h1>;
+
+  return (
+    <>
+      <h1>WTF? What to food</h1>
+      <h3>my super tasty selection of recipes:</h3>
+
+      <recipeList>
+        {data?.map((data) => (
+          <FoodCard />
```

Comment:
You have to pass the data as a prop to your `FoodCard`, otherwise the data cannot be displayed.

```suggestion
        {recipes?.map((recipe) => (
          <FoodCard recipe={recipe}/>
```